backend/CircleForms/Controllers/V1/PostsController.cs

```diff
@@ -125,6 +128,25 @@ public async Task<IActionResult> Answer(string id, [FromBody] List<SubmissionCon
         return Ok();
     }
 
+    private static string GenerateAccessKey(byte size)
```

Comment:
Might wanna extract that from the controller to somewhere else

Reply:
i will do this right not but not in this pr